systems.csv

```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
 BR,Bike VV,"Vila Velha, BR",bike_vv,https://www.bikevv.com.br/,https://vilavelha.publicbikesystem.net/ube/gbfs/v1/
 CA,Bike Share Toronto,"Toronto, ON",bike_share_toronto,https://www.bikesharetoronto.com/,https://tor.publicbikesystem.net/ube/gbfs/v1/
 CA,BIXI-Montreal,"Montreal, QC",Bixi_MTL,https://www.bixi.com/,https://api-core.bixi.com/gbfs/gbfs.json
+CA,Bunny,"Kelowna, BC",bunny_kelowna,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike//gbfs/en/gbfs.json
```

Comment:
```suggestion
CA,Bunny,"Kelowna, BC",Bunny,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike/gbfs/en/gbfs.json
```
Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.